src/components/products/Products.js

```diff
@@ -1,14 +1,19 @@
 import "./products.css";
 import { Rating } from "../index";
-import { useProduct, useLogin, useCart } from "../../context/index";
+import {
+  useProduct,
+  useLogin,
+  useCart,
+  useWishlist,
+} from "../../context/index";
```

Comment:
no need to add index at last it takes by default

Reply:
Thanks. I am unaware about this